TwilioVerifyDemo/TwilioVerifyDemo.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 52;
+	objectVersion = 54;
```

Comment:
@AlejandroOrozco Please update the supported versions in the readme for Xcode and Swift